README.md

```diff
@@ -144,24 +161,31 @@ The returned object from an execute call possesses the following methods:
 }
 ```
 
-#### Usage in a Web Browser
+### Use in a web browser
 
-The signalflow client can be built for usage in a browser. This is accomplished via the following commands.
+The signalflow client can be built for use in a browser. 
 
-```
-$ npm install
-$ npm run build:browser
-The output can be found at ./build/signalflow.js
-```
+1. Run the following commands:
 
-It can then be loaded as usual via a script tag
+   ```
+   npm install
+   npm run build:browser
+   ```
 
-```
-<script src="build/signalflow.js" type="text/javascript"></script>
-```
+   Expected output:
+
+   ```
+   The output can be found at ./build/signalflow.js
+   ```
+
+2. Load the client using a `script` tag:
+
+   ```
+   <script src="build/signalflow.js" type="text/javascript"></script>
+   ```
 
-Once loaded, signalfx global will be created(window.signalfx). Note that only the SignalFlow package is included in this built file.
+   Once loaded, a global ``signalfx`` object is available in the browser. Note that the built file only includes the SignalFlow package.
```

Comment:
@seemk Just want to confirm that this works.

Reply:
Seems good!